src/bin/etl_twizzler/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "etl_twizzler"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[target.'cfg(target_os = "twizzler")'.dependencies]
+twizzler-abi = { path = "../../lib/twizzler-abi", features = ["runtime"] }
```

Comment:
Are relative paths the right way to do this?  @dbittman can you comment here?

Reply:
For now I think it's fine. We're only intending this package to be built in this repo, so it should be okay.

We will eventually want to fix these, but that'll require a whole restructuring of the repo, and more official rust support for twizzler...